presto-main/src/main/java/com/facebook/presto/operator/ExchangeClientConfig.java

```diff
@@ -141,4 +142,16 @@ public ExchangeClientConfig setAcknowledgePages(boolean acknowledgePages)
         this.acknowledgePages = acknowledgePages;
         return this;
     }
+
+    @Config("exchange.response-size-exponential-moving-average-decaying-alpha")
+    public ExchangeClientConfig setAverageBytesEMAAlpha(double averageBytesEMAAlpha)
```

Comment:
please rename getter/setter to match the variable name

Reply:
Renamed